containers/Makefile

```diff
@@ -56,28 +56,42 @@ ifndef IMAGE_PREFIX
 endif
 $(info IMAGE_PREFIX=$(IMAGE_PREFIX))
 
-
-all: nginx-proxy nginx-storage concent-api postgresql verifier
-
-nginx-proxy: build/ $(NGINX_PROXY_SOURCE) build/nginx-proxy/concent-api-assets.tar build/nginx-proxy/concent-api-docs.tar
-	docker build --tag $(IMAGE_PREFIX)nginx-proxy:$(IMAGE_VERSION) build/nginx-proxy/
-	docker tag $(IMAGE_PREFIX)nginx-proxy:$(IMAGE_VERSION) $(IMAGE_PREFIX)nginx-proxy:latest
-
-nginx-storage: build/ $(NGINX_STORAGE_SOURCE)
-	docker build --tag $(IMAGE_PREFIX)nginx-storage:$(IMAGE_VERSION) build/nginx-storage/
-	docker tag $(IMAGE_PREFIX)nginx-storage:$(IMAGE_VERSION) $(IMAGE_PREFIX)nginx-storage:latest
-
-postgresql: build/ $(POSTGRESQL_SOURCE)
-	docker build --tag $(IMAGE_PREFIX)postgresql:$(IMAGE_VERSION) build/postgresql/
-	docker tag $(IMAGE_PREFIX)postgresql:$(IMAGE_VERSION) $(IMAGE_PREFIX)postgresql:latest
-
-concent-api: build/ $(CONCENT_API_SOURCE) build/concent-api/requirements.lock build/concent-api/testing.pref build/repositories/concent/
-	docker build --tag $(IMAGE_PREFIX)concent-api:$(IMAGE_VERSION) build/concent-api/
-	docker tag $(IMAGE_PREFIX)concent-api:$(IMAGE_VERSION) $(IMAGE_PREFIX)concent-api:latest
-
-verifier: build/ $(VERIFIER_SOURCE) build/repositories/golem/ build/verifier/golem/scripts/ build/verifier/golem/imgverifier-requirements.txt
-	docker build --tag $(IMAGE_PREFIX)verifier:$(IMAGE_VERSION) build/verifier/
-	docker tag $(IMAGE_PREFIX)verifier:$(IMAGE_VERSION) $(IMAGE_PREFIX)verifier:latest
+IMAGE_TARGETS := 	\
+    nginx-proxy  	\
+		nginx-storage	\
+    postgresql		\
+		concent-api 	\
+		verifier 			\
+
+
+nginx-proxy!dependencies:                    					\
+    $(NGINX_PROXY_SOURCE)                    					\
+    build/nginx-proxy/concent-api-assets.tar 					\
+    build/nginx-proxy/concent-api-docs.tar
+
+nginx-storage!dependencies:														\
+		$(NGINX_STORAGE_SOURCE)
+
+concent-api!dependencies:               							\
+    $(CONCENT_API_SOURCE)               							\
+    build/concent-api/requirements.lock 							\
+    build/concent-api/testing.pref      							\
+    build/repositories/concent/
+
+postgresql!dependencies: 															\
+    $(POSTGRESQL_SOURCE)
+
+verifier!dependencies:																\
+		$(VERIFIER_SOURCE) 																\
+		build/repositories/golem/ 												\
+		build/verifier/golem/scripts/ 										\
```

Comment:
Please fix the whitespace here.

- You're inserting tabs everywhere. Even inside the text (e.g. `build/verifier/golem` and `\`). We're using spaces everywhere, please use them too. Otherwise the code looks very misaligned.
    - The only place where we use tabs is at the beginning of a line containing a bash command in a target. And that's only because make syntax requires it.
- It's fine to add a little whitespace before `\` but this is way too much. There's so much that I have to scroll the diff horizontally to see that there's `\` at the end of some lines.
- `\` should all be either in the same column (preferred) or immediately after the text.

I have just realized that we don't have this written down in our style yet so I have added a new proposal. Please see https://github.com/Code-Poets/coding-style/issues/16

Reply:
yeah, the problem is with different rendering engine in github and atom, apparently tabs in atom does not have fixed length, and slashes appeared to be alligned, definetely within screen width

Reply:
excessive tabs replaced with spaces